packages/client/hmi-client/src/temp/tera-simulation-workflow.vue

```diff
@@ -1,22 +1,86 @@
 <template>
-	<infinite-canvas debug-mode :last-transform="lastTransform" @save-transform="saveTransform">
+	<infinite-canvas
+		debug-mode
+		@contextmenu="toggleContextMenu"
+		:last-transform="canvasTransform"
+		@save-transform="saveTransform"
```

Comment:
There is a bit of a self-referential logic here that we should try to untangle.
- canvasTransform is sent as a prop into the component
- saveTransform modifies canvasTransform 
- canvasTransform is sent as a prop into the component ... etc etc

I think we should just take out `:last-transform` for now to avoid some inadvertent watcher going to infinite loops